content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,124 @@
+
+# Key Features
+
+## Let your content shine like never before
+
+If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted to this so it seems they wanted to:
+
+#### Jellyfin 10.4.3
+
+<img src="/images/10.5-release/old.png" name="Old item details page" />
+
+#### Jellyfin 10.5
+
+<img src="/images/10.5-release/new.png" name="New item details page" />
+
+*With 10.5, we wanted to reimagine Jellyfin's Web interface*
+
+Posters are now as relevant as the description of the element and backgrounds introduce you to your content. In fact, I thought that I was in the jungle when
+I took that screenshot of the *Jungle's Book* wallpaper. Do you also feel it?
+
+Also notice that the header of the UI has been reworked to have a *slightly* lighter header
+
+Shoutout to all the Web team for their invaluable contributions in mking this happen, specially to *[MrTimscampi](https://github.com/mrtimscampi)* and 
+*[grafixeyehero](https://github.com/grafixeyehero)*.
+
+You can try out the redesign on our [demo server](https://demo.jellyfin.org/stable)
+
+## Faster than the speed of sound and cleaner than a surgery room
+
+As you might know, we are still a relatively young community and our goal is to provide the best self-hosted media server out there. Like usual, we did what we can to clean up the code and a lot of this work will pass unnoticed to many of you, but they're not less important. I'll get to that in a moment.
+
+
+Let's start with the web client, since it's the most obvious to you. A lot of work has been done to simplify the code and ease the transition
+to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. While
+all the benefits of this are not visible for you in the daily use just yet, it's a necessary step towards a better application. In fact, it already paid off in application speed. See this video:
+
+<video autoplay muted loop preload="none" id="preview">
+    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
+</video>
+
+Not only it is faster, we have new animations too! Images will fade in every time you change pages. If you don't like the new animations they can be modified in settings
+
+We finally fixed the long standing issue many of you had with images (which I introduced, didn't mean to cause it!). Although the issue is no longer around us, we still aren't happy with the end result and hope to further improve it going forward.
+The long-standing issue that many of you had ( of this post 🙄) caused by the images has been fixed in this version
+(although we want to keep improving in that matter, as we're still not completely happy with the current implementation) 
```

Comment:
This sentence is duplicated.

Reply:
True, forgot to remove the old one, thanks for pointing that out 😅.